postgresql-plugin/src/main/java/co/cask/postgres/PostgresAction.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.postgres;
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Name;
+import co.cask.cdap.api.annotation.Plugin;
+import co.cask.cdap.etl.api.action.Action;
+import co.cask.db.batch.action.AbstractDBAction;
+import co.cask.db.batch.config.DBSpecificQueryConfig;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+/**
+ * Action that runs PostgreSQL command.
+ */
+@Plugin(type = Action.PLUGIN_TYPE)
+@Name(PostgresConstants.PLUGIN_NAME)
+@Description("Action that runs a MySQL command")
```

Comment:
MySQL -> PostgreSQL (same for other plugins)

Reply:
done